doc/source/user_guide/post.rst

```diff
@@ -23,6 +23,83 @@ streamed back to the client for review or visualization.
    via a Python client.
 
 
+
+Enriched Command Output
+~~~~~~~~~~~~~~~~~~~~~~~
+All :class:`Mapdl <ansys.mapdl.core.mapdl._MapdlCore>` commands output a string object which can be parsed to 
+obtain specific data from it.
+
+In certain :class:`Mapdl <ansys.mapdl.core.mapdl._MapdlCore>` commands the returned string contains 
+some methods to process the output.
+These commands are listed in Table-1_.
+
+.. _Table-1:
+
+**Table 1. Commands with extra processing methods in the output.**
+
++----------------+---------------------------------------------------------------------------------------------------+----------------------------------------------------------+
+| Category       | Extra Methods Available                                                                           | Mapdl Commands                                           |
++================+===================================================================================================+==========================================================+
```

Comment:
I hope you didn't have to do this by hand. Note that there are tools like https://www.tablesgenerator.com/text_tables.

Reply:
I know the page. I use sublime text for generating most of the tables.